packages/next/src/server/base-server.ts

```diff
@@ -702,6 +702,15 @@ export default abstract class Server<ServerOptions extends Options = Options> {
   ): Promise<void> {
     await this.prepare()
     const method = req.method.toUpperCase()
+
+    // Extract the w3c trace context headers and pass them to tracer
+    const traceParentHeader = req.headers.traceparent
+    const traceStateHeader = req.headers.tracestate
+    const traceParent =
```

Comment:
If we extract the propagation, should we also inject it in the downstream fetch calls? Or would that be too subjective?